### DISCUSSION TITLE
Welcome to rock-4se-kali Discussions!

### DISCUSSION THREAD

**Discussion by byte4RR4Y** [byte4RR4Y/rock-4se-kali#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by byte4RR4Y** [comment]
fell free to offer ideas or reporting any issues.

**Comment by primearcanum** [comment]
Hi, I am quite new to this community and I just received my rock 5b SBC, I can't seem to boot this off of my Rock 5b, 
I am running it on an SD card.
Also confused on that "Mount Root File System".

Thank you!

**Comment by Millyvanilly1988** [comment]
Hi!
Does it work with rock pi 4 c+ ?
